akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMap.scala

```diff
@@ -33,12 +33,10 @@ object ORMap {
  * This class is immutable, i.e. "modifying" methods return a new instance.
  */
 @SerialVersionUID(1L)
-final class ORMap[A <: ReplicatedData] private[akka] (
+final class ORMap[A <: AbstractReplicatedData[_]] private[akka] (
```

Comment:
yeah, in other places as well, I'm helping out on getting this PR up to type-soundness right now :-)